verisim/Makefile

```diff
@@ -21,19 +21,22 @@ LDFLAGS := $(LDFLAGS) -L$(RISCV)/lib -Wl,-rpath,$(RISCV)/lib -L$(sim_dir) -lfesv
 include $(base_dir)/Makefrag
 include $(sim_dir)/Makefrag-verilator
 
+sim_blackboxes = \
+  $(build_dir)/firrtl_black_box_resource_files.f
 rocketchip_vsrc_dir = $(ROCKETCHIP_DIR)/src/main/resources/vsrc
```

Comment:
Can we add a blank line between `sim_blackboxes` and `rocketchip_vsrc_dir` (or put `sim_blackboxes` on one line) to match style?

Reply:
The problem with using csrc in testchipip is that I'd like this to be able to work without having the submodule there. Maybe that's silly, but I'm looking at the notebooks as a use case.

There's a way to make this work with the blackbox stuff, but I'm not sure how to make it work because they don't really belong to a blackbox. I'm thinking of the best way to make it work

Reply:
The problem with using csrc in testchipip is that I'd like this to be able to work without having the submodule there. Maybe that's silly, but I'm looking at the notebooks as a use case.

There's a way to make this work with the blackbox stuff, but I'm not sure how to make it work because they don't really belong to a blackbox. I'm thinking of the best way to make it work

Reply:
Does the requisite tooling live in project-template for it to work without testchipip? Or is this a packaging issue? Can you publish them in src/main/resources and then unzip them from jar in project-template?